tests/ConnectionTest.php

```diff
@@ -32,6 +32,15 @@ public function testDb()
         $this->assertInstanceOf(\MongoDB\Client::class, $connection->getMongoClient());
     }
 
+    public function testDsnDb()
+    {
+        $connection = DB::connection('dsn_mongodb_db');
+        $this->assertInstanceOf(\MongoDB\Database::class, $connection->getMongoDB());
+
+        $connection = DB::connection('dsn_mongodb_db');
+        $this->assertInstanceOf(\MongoDB\Client::class, $connection->getMongoClient());
```

Comment:
Leave only `Client::class` please

Reply:
I've just copied from upper function, see https://github.com/jenssegers/laravel-mongodb/blob/master/tests/ConnectionTest.php#L26

Should we refactor all tests? It's not like me who did that 😉

Reply:
OK, I suggest refactor tests in another PR